### PR TITLE
Create AutomaticScenarioAssignement migrations and entity

### PR DESCRIPTION
**Pull request by kjaksik**
**Description**

Changes proposed in this pull request:
- create migrations
- create entity struct

**Related issue(s)**
#1024

### CODE REVIEW

File: components/schema-migrator/migrations/director/202003231306_add_automatic_scenario_assignement.up.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE automatic_scenario_assignments (
+    scenario VARCHAR(128),
+    tenant_id UUID NOT NULL,
+    key VARCHAR(256) NOT NULL,
+    value JSONB);
```
Comment (aszecowka) [inline components/schema-migrator/migrations/director/202003231306_add_automatic_scenario_assignement.up.sql]: I am wondering if the value should be of type JSONB, or just pure string. We discussed that we are going to support only simple values. @crabtree what do you think?

Comment (crabtree) [reply components/schema-migrator/migrations/director/202003231306_add_automatic_scenario_assignement.up.sql]: I'd go with simple string here.

File: components/director/internal/domain/scenarioassignement/entity.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package scenarioassignement
+
+type Entity struct {
+	Scenario string `db:"scenario"`
+	TenantID string `db:"tenant_id"`
+	Key      string `db:"key"`
```
Comment (crabtree) [inline components/director/internal/domain/scenarioassignement/entity.go]: How about:
```suggestion
	SelectorKey      string `db:"selector_key"`
```

File: components/director/internal/domain/scenarioassignement/entity.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+	Value    string `db:"value"`
```
Comment (crabtree) [inline components/director/internal/domain/scenarioassignement/entity.go]: How about:
```suggestion
	SelectorValue    string `db:"selector_value"`
```

**Pull request merged**